tests/psoc6/pin.py

```diff
@@ -0,0 +1,24 @@
+from machine import Pin
+
+# Tests for Output pin
+# Instantiate output pin for led operation
+# Constr
+p1 = Pin("P13_7")
+p1.init(Pin.OUT)
+print(p1)  # Pin:111 or P13_7, Mode=OUT, Pull=None, Value=1
+
+p1.value(0)  # Led glows
```

Comment:
I don´t want to complicate much more the setup for now, but we could connect 2 pins together and actually check the input against an output automatically. LED would require visual inspection.
That will require a HIL hardware setup documented somewhere.

Reply:
Yeah i agree.